dask_ml/model_selection/_incremental.py

```diff
@@ -757,7 +859,58 @@ def _get_params(self):
         else:
             return ParameterSampler(self.parameters, self.n_initial_parameters)
 
+    def _stop_on_plateau(self, info, instructions):
+        _best_plateau = {}
```

Comment:
I'm probably missing something, but can you walk through how a model with `patience=0` works? From what I can see, it looks like all the updates to `_best_plateau` are done inside `if self.patience` blocks. So for no patience you'll never update `_best_plateau`.

If you never update `_best_plateau` here, then you'll return an empty set to `_additional_calls`.

Ah, but I see now that `_additional_calls` falls back on the instructions of `out` and `steps`, so hopefully that takes care of decaying us to 0 additional fit calls. Is that correct? I'll leave this comment here, but I think we're OK.

Reply:
I think your interpretation is correct. Most of it relies on 

> that `_additional_calls` falls back on the instructions of out and steps